src/main/java/ai/rev/speechtotext/models/CustomerUrlData.java

```diff
@@ -0,0 +1,42 @@
+package ai.rev.speechtotext.models;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.Map;
+
+public class CustomerUrlData {
```

Comment:
Whole class is indented too far. Seems like standard in this repo is 2 spaces indents

Reply:
oh yeah.... I think that's very non-standard for java but i'll keep it consistent